ClearKeep/Scenes/Profile/Views/CountryCode.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  CountryCode.swift
+//  ClearKeep
+//
+//  Created by đông on 14/03/2022.
+//
+
+import SwiftUI
+import Combine
+import Common
+import CommonUI
+
+private enum Constant {
+	static let spacer = 25.0
+	static let paddingVertical = 14.0
+	static let paddingHorizontal = 24.0
+}
+
+struct CountryCode: View {
+	let inspection = ViewInspector<Self>()
+	@Environment(\.injected) private var injected: DIContainer
+	@Environment(\.colorScheme) private var colorScheme
+	@Environment(\.presentationMode) private var presentationMode: Binding<PresentationMode>
+
+	@Binding var isShowing: Bool
+	@State private(set) var samples: Loadable<[IProfileModel]>
+	@State private(set) var search: String
+	@State private(set) var searchStyle: TextInputStyle = .default
+	@State private(set) var isShowUserProfile = false
+	@Binding private(set) var selectedNum: String
+
+	init(selectedNum: Binding<String>, isShowing: Binding<Bool>, samples: Loadable<[IProfileModel]> = .notRequested,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/1ab8931f0f28255ebeb64e047f8b06b326be6a2d/ClearKeep/Scenes/Profile/Views/CountryCode.swift#L32"></span>Functions and methods parameters should be either on the same line, or one per line.<br />
<code>multiline_parameters</code> <code>CountryCode.swift:32</code></td>
    </tr>
  </tbody>
</table>